constants.js

```diff
@@ -86,6 +86,16 @@ if (!abilities.nevermore_shadowraze3)
   // Find and replace short raze range with long raze range
   abilities.nevermore_shadowraze3.attrib = abilities.nevermore_shadowraze3.attrib.replace(/\d{3}/, 700);
 }
+if (!abilities.beastmaster_call_of_the_wild_boar)
+{
```

Comment:
A simpler fix might be just to use JSON.parse on the JSON you put in the other file, then assign it to abilities.beastmaster_call_of_the_wild_boar